src/main/kotlin/no/nav/syfo/personstatus/api/v2/endpoints/PersonOversiktStatusApiV2.kt

```diff
@@ -91,5 +92,19 @@ fun Route.registerPersonoversiktApiV2(
                 call.respond(HttpStatusCode.BadRequest, e.message ?: "Kan ikke hente personoversikt for enhet")
             }
         }
+
+        get("/personer/single") {
```

Comment:
Usikker på hvordan vi skal navngi denne for å signalisere at vi henter en ressurs og ikke flere🤔 Hadde ideelt sett blitt seende slik ut:
```
"/personer/{person-id}"
```

Reply:
Tror dette egentlig hører mer hjemme i `PersonTildelingApiV2`enn i oversikstatus-apiet. Kan utvides med GET-endepunkt for å hente veileder-knytning for en person. Der har vi allerede endepunkter for å hente knytninger for en veilder og endre knytninger. Og så trengs det tilgangskontroll som sjekker at veileder har tilgang til person :)

Reply:
Hm, har tenkt at siden domeneobjektet og ressursen er den samme så vil man nå det ved å bruke det samme API'et 🤔 Men her er kanskje 2 konsepter litt bakt inn i samme ressurs? Status på personen i forhold til oppgaver/hendelser og tilknytning? Kan flytte til `PersonTildelingApiV2`

Reply:
Ja, er nok ikke helt rendyrket konsept per ressurs i denne appen, men hvis vi skal følge inndelingen av APIer vi har her så er det logisk å ha det i `PersonTildelingApiV2` og at man returnerer en `VeilederBrukerKnytning` i responsen.